deployment/ansible/roles/nyc-trees.app/templates/upstart-nyc-trees-app.conf.j2

```diff
@@ -11,4 +11,4 @@ respawn
 setuid nyc-trees
 chdir {{ app_home }}
 
-exec envdir /etc/nyc-trees.d/env gunicorn --config /etc/nyc-trees.d/gunicorn.py nyc_trees.wsgi
+exec envdir /etc/nyc-trees.d/env gunicorn --config /etc/nyc-trees.d/gunicorn.py nyc_trees.wsgi >> {{ app_log }} 2>&1
```

Comment:
Double `>` right there is a syntax error. The `>>` right after the `wsgi` file handles making sure we append to the file vs. overwriting it.